pkg/util/util.go

```diff
@@ -14,6 +14,7 @@ import (
 	goVersion "github.com/hashicorp/go-version"
 )
 
+// UTILS CAN GO HERE
```

Comment:
nit: code cleanup
```suggestion
```

Reply:
my bad - reminder to myself during a convo with @sunkickr that snuck in :/ didn't intend to commit